docker/docker_image_src.go

```diff
@@ -39,6 +39,19 @@ func newImageSource(ctx *types.SystemContext, ref dockerReference, requestedMani
 	if requestedManifestMIMETypes == nil {
 		requestedManifestMIMETypes = manifest.DefaultRequestedManifestMIMETypes
 	}
+	var (
+		mts   = supportedManifestMIMETypesMap()
+		reset = true
```

Comment:
…also, no need for the `var()` here AFAICS, just use `:=`. Or does that have an idiomatic meaning?